<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialTorPlot</class>
 <widget class="QDialog" name="DialTorPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>256</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Torque Sensor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0" colspan="5" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Unit Type</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QCheckBox" name="nmCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>N.m</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="mvCheck">
     <property name="text">
      <string>mV</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_3</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QCheckBox" name="nmmCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>N.mm</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="2" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="vCheck">
     <property name="text">
      <string>V</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_3</string>
     </attribute>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="yminLabel_2">
       <property name="text">
        <string>Ymin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="yminLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ymaxLabel_2">
       <property name="text">
        <string>Ymax</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ymaxLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stepSizeLabel_3">
       <property name="text">
        <string>Step size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="stepSizeYLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="xminLabel_2">
       <property name="text">
        <string>Xmin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="xminLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="xmaxLabel_2">
       <property name="text">
        <string>Xmax</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="xmaxLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stepSizeLabel_4">
       <property name="text">
        <string>Step size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="stepSizeXLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="5" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Axes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QRadioButton" name="physButton">
     <property name="text">
      <string>Physical</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QRadioButton" name="vButton">
     <property name="text">
      <string>Voltage</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vButton</tabstop>
  <tabstop>vCheck</tabstop>
  <tabstop>mvCheck</tabstop>
  <tabstop>physButton</tabstop>
  <tabstop>nmCheck</tabstop>
  <tabstop>nmmCheck</tabstop>
  <tabstop>xminLineEdit</tabstop>
  <tabstop>xmaxLineEdit</tabstop>
  <tabstop>stepSizeXLineEdit</tabstop>
  <tabstop>yminLineEdit</tabstop>
  <tabstop>ymaxLineEdit</tabstop>
  <tabstop>stepSizeYLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
